Formula/par.rb

```diff
@@ -15,7 +15,7 @@ class Par < Formula
 
   conflicts_with "rancid", because: "both install `par` binaries"
 
-  # A patch by Jérôme Pouiller that adds support for multibyte
+  # A patch by Jerome Pouiller that adds support for multibyte
```

Comment:
👍 to just removing the name. This is definitely bikeshedding, for what it's worth so I'm going to just pick one then merge.